Reminder-App/Reminder-App/ReminderDetailEditDataSource.swift

```diff
@@ -9,63 +9,72 @@ import UIKit
 
 class ReminderDetailEditDataSource: NSObject {
     
+    typealias ReminderChangeAction = (Reminder) -> Void
+    
     enum ReminderSection: Int, CaseIterable {
-        
         case title
         case dueDate
         case note
         
         // MARK: Public Methods
         func cellIdentifier(for row: Int) -> String {
-            
             switch self {
             case .title:
                 return "EditTitleCell"
```

Comment:
You have extended the UITableViewCell with a computed property i.e reusableIdentifier to return the identifier, then use that property instead of using "EditTitleCell"